src/transactions/transactionBytes.js

```diff
@@ -12,25 +12,17 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-/**
- * Crypto module provides functions for byte/fee calculation, hash/address/id/keypair generation,
- * plus signing and verifying of transactions.
- * @class crypto
- */
 /* eslint-disable no-plusplus */
-import crypto from 'crypto-browserify';
```

Comment:
This library is just use once in the whole project
https://github.com/LiskHQ/lisk-js/blob/8e7d374881a47cf9baaa4afb9ebe55f81609e8ed/src/utils/mnemonic.js#L31
I suggest to find an alternative and erase it